api/src/main/java/org/pmiops/workbench/concept/ConceptService.java

```diff
@@ -223,6 +218,21 @@ public ConceptIds classifyConceptIds(Set<Long> conceptIds) {
     return new ConceptIds(standardConceptIds.build(), sourceConceptIds.build());
   }
 
+  public static Concept toClientConcept(DbConcept dbConcept) {
+    boolean standard = STANDARD_CONCEPT_CODES.contains(dbConcept.getStandardConcept());
+    return new Concept()
```

Comment:
Is there a ticket to move this to mapper?

Reply:
My plan is to eventually do away with this ConceptService once we migrate everything over to cohort builder search. We have a ticket to remove it here: https://precisionmedicineinitiative.atlassian.net/browse/RW-5600

Reply:
When migrated to CB search we should only be returning Criteria objects and will no longer need the Concept object